apps/ergw_core/src/ergw_core_config.erl

```diff
@@ -36,7 +36,21 @@ to_map(L) when is_list(L) ->
 	      erlang:error(badarg, [Opt])
       end, #{}, normalize_proplists(L)).
 
-get([Key|Next], Default) ->
+get([Key|_] = Query, Default) ->
+    case is_global_key(Key) of
+	true  -> get_global_key(Query, Default);
+	false -> get_local_key(Query, Default)
+    end.
+
+get_global_key(Query, Default) ->
+    case ergw_global:find(Query) of
+	{ok, Value} ->
+	    {ok, Value};
+	_ ->
+	    {ok, Default}
```

Comment:
It would be informative if in the log will provided information that was provided the default value was taken.

Reply:
why? there could be thousands of those case, that would only spam the log.